src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+
+/**
+ * Main class to run Ui object and listen for user input.
+ */
+public class Duke {
+    private TaskList tasks;
+    private Ui ui;
+    private Storage storage;
+
+    /**
+     * Constructor to initialise Duke object.
+     * @param filePath Location of Duke.ser
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.loadData());
+        } catch (Exception e) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    /**
+     * You should have your own function to generate a response to user input.
+     * Replace this stub with your completed method.
+     */
```

Comment:
Not too sure what this comment is about.

Reply:
Sorry my bad. Forgot to change the template javadoc. Thank you for pointing it out.